src/main/webapp/app/tasks/task-search/task-search.component.ts

```diff
@@ -0,0 +1,45 @@
+import { Component, EventEmitter, OnDestroy, OnInit, Output } from '@angular/core';
+import { FormControl, FormGroup, Validators } from '@angular/forms';
+import { LocalTaskService } from 'app/tasks/local-task.service';
+import { Subscription } from 'rxjs';
+import { distinctUntilChanged } from 'rxjs/operators';
+
+@Component({
+  selector: 'tiny-task-search',
+  templateUrl: './task-search.component.html',
+  styleUrls: ['./task-search.component.scss']
```

Comment:
would be nice to use the change detection strategy `On.Push`

Reply:
But here I don't have any Inputs so I don't see any reason for On.Push